packages/adaptive_scaffold/pubspec.yaml

```diff
@@ -0,0 +1,56 @@
+name: adaptive_scaffold
+description: This package contains some helper widgets that make the process of developing adaptive layouts easier, especially with navigational elements.
+version: 0.0.1
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3Aadaptive_scaffold
+repository: https://github.com/flutter/packages/adaptive_scaffold
+homepage:
+
+environment:
+  sdk: '>=2.17.0 <3.0.0'
+  flutter: ">=1.17.0"
```

Comment:
Please don't use a minimum version older than 2.8, as that's the earliest we do any automated tests for in CI.

In practice, the Dart SDK version above means you are restricting to Flutter 3.0, so it would be less confusing to just say that.